gateway/ln-gateway/src/error.rs

```diff
@@ -55,6 +56,12 @@ impl IntoResponse for PublicGatewayError {
             ),
         };
 
+        let error_message = if std::env::var(FM_DEBUG_GATEWAY_ENV).is_ok() {
```

Comment:
[nit] There's a helper `is_env_var_set` that can be useful for boolean-like env vars since you can also set them to `0` or `false` to unset.

https://github.com/fedimint/fedimint/blob/c22a0b017d47824a076d88a32a9ab80cc4f2818a/fedimint-core/src/envs.rs#L19

Reply:
Fixed!